std/base64.d

```diff
@@ -1683,10 +1683,28 @@ template Base64Impl(char Map62th, char Map63th, char Padding = '=')
      * }
      * -----
      */
+    auto decoder()(const(char)[] range)
```

Comment:
Is there a reason, why the return type is `auto` and not `Decoder!(const(ubyte)[])`?

Reply:
The reason is that I initially used `.byCodeUnit`, and there was no good way to mark the return type. But now when it's `.representation` instead there's no good reason anymore. Will change.